docs/success-stories/success-story-1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: success-story-page
+show_sidebar: false
+permalink: /success-stories/success-story-The-Digital-Slide-Archive-and-TCGA/
+
+# Banner Content
+# =================================
+title: Success Stories
+subtitle: The Digital Slide Archive and TCGA
+hero_image: /assets/img/home-callouts/success_stories.jpg
+
+# Story
+# =================================
+story:
+ - title: The Digital Slide Archive and TCGA
+   link-id: The-Digital-Slide-Archive-and-TCGA
+   description-para-1: "The Cancer Digital Slide Archive is a website that hosts whole-slide images of pathologic specimens associated with the TCGA studies. At over 10,000 images and 20 Terabytes, the CDSA has enabled a number of the TCGA Analysis Working Group Studies [sources], as well as other investigator-driven studies that utilize the digital pathology images [sources]<br><br>
+   The Digital Slide Archive project builds on the success of our experience with the Cancer Digital Slide Archive, creating a software platform that allows users to create their own digital pathology servers to support data management and hosting for their own investigations.  Comments about who else is using this."
```

Comment:
@dgutman, @cooperlab We need to fill this out (or remove the phrase "Comments about who else is using this.").

Reply:
For now I have removed 'comments about who else is using this' in 683f1fdf63e806d1c2cadadc736f3e3415eab960
let me know if there is something to add here.

Reply:
@dgutman If we have a nice list of users and their links or testimonials, I think this could even exist as a separate tab on the main page instead of buried here.